src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.js

```diff
@@ -181,18 +184,6 @@ const SplitForm = ({ incident, attachments, onHandleCancel, onHandleSubmit }) =>
   );
 };
 
-SplitForm.defaultProps = {
-  incident: {
-    category: {
-      category_url: '',
-    },
-    priority: {
-      priority: '',
-    },
-  },
-  attachments: [],
-};
-
 SplitForm.propTypes = {
   incident: incidentType,
```

Comment:
Shouldn't this prop be required? Since you removed the default prop.

Reply:
default value is now provided